GPy/core/sparse_gp_mpi.py

```diff
@@ -88,9 +88,9 @@ def optimizer_array(self, p):
     def optimize(self, optimizer=None, start=None, **kwargs):
         self._IN_OPTIMIZATION_ = True
         if self.mpi_comm==None:
-            super(SparseGP_MPI, self).optimize(optimizer,start,**kwargs)
+            ret = super(SparseGP_MPI, self).optimize(optimizer,start,**kwargs)
```

Comment:
@mzwiessele, sorry that I didn't see this. It looks OK to me. It would not change the behavior of MPI.